.github/workflows/docker-image-release.yml

```diff
@@ -29,19 +29,19 @@ jobs:
           images: |
             jetpackio/devbox
           tags: |
-            type=semver,pattern={{version}}
+            type=semver,pattern={{version}},value=${{ inputs.tag }}
           flavor: |
-            latest=${{ inputs.tag && true || false }}
+            latest=${{ !inputs.tag && true || false }}
       - name: Docker meta root
         id: metaroot
         uses: docker/metadata-action@v5
         with:
           images: |
             jetpackio/devbox-root-user
           tags: |
-            type=semver,pattern={{version}}
+            type=semver,pattern={{version}},value=${{ inputs.tag }}
           flavor: |
-            latest=${{ inputs.tag && true || false }}
```

Comment:
Did you ever try removing this (which would cause it to default do `latest=auto`?

The current setting (before and after this PR) are both wrong. It would be possible to create a tag that is not latest and we would certainly not want to update the `latest` image in that case.

Reply:
I looked into documentation for `auto`. It adds latest tag if the code is from the `master` branch. There is also a setting to configure `tag` to have `latest` value if the code is on the default branch (`main` in our case). Both cases don't really apply to our use case. We want to have latest when the workflow is triggered by a tag creation and not have latest when we put a custom tag in the input and trigger the workflow manually.
Initially, I had 2 different yaml files for them but you suggested merging them since they were very similar. 
How is the setting in this PR wrong? it's saying if `input.tag` is not available (meaning the workflow was triggered by a tag creation event) latest is true. Which is what we want.